### ISSUE TITLE
write branches

### ISSUE THREAD

**Issue by LuridBolete** [CamWagner/lizzie#462]
Thanks to developers! I love Lizzie. I would like a new option - LZ to write in the sgf file branches, which it considers the best.

**Comment by featurecat** [comment]
Me too! I think for next version I'll add this. Maybe during January? Thanks :)

**Comment by lephis** [comment]
> 
> 
> Me too! I think for next version I'll add this. Maybe during January? Thanks :)

I'm eager to have this function too! Especially saving to SGF file 3 or 5 best variations (+their win rate) of every step during automatic analysis. 

With this function implemented, I could start a deep automatic analysis before going to bed; and "harvest" the full result after waking up  :D

**Comment by Phelanpt** [comment]
This would be great! :) I currently don't have access to a gpu version, so I do analysis with the cpu only lz, and it often goes overnight for a decent analysis. :)
I'd suggest that you look at how GoReviewPartner is working, and at its rsgf format; and also at how Sabaki is saving win probabilities through sgf properties. Not sure if their methods are the best, but could be helpful to standardise the format somehow. :)

**Comment by lephis** [comment]
> 
> 
> I also like to run overnight reviews. That's why I'm so excited!
> 
> Another exciting feature to add: instead of specifying a number of playouts per move, what about optimizing to run analysis for a specific amount of time? Lizzie could determine how many playouts to devote to each move.

Yeah, great idea, I love it! Ideally Lizzie can provide two exclusive options: Playout number or Timer. It's up to user to make his choice. (It seems to me that Chinese pro players would always prefer a specific playout number, in order to make sure the analysis quality.)
